lib/console.js

```diff
@@ -139,12 +139,14 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
-Console.prototype.time = function time(label) {
+Console.prototype.time = function time(label = 'default') {
```

Comment:
This is okay because of the method binding in [L62](https://github.com/jasnell/node/blob/7488fde5b34a4c27394569955f6c789c91d40a23/lib/console.js#L62)?